docker/postgres/pubminer-schema.sql

```diff
@@ -0,0 +1,12 @@
+-- Initializes the pubminer schema. This is a stub schema and should be replaced with the
+-- schema definition
+
+CREATE TABLE demographic_index(
```

Comment:
@garzillo we should replace the contents of this sql file with whatever schema you came up with for the DB